hexa/user_management/tests/test_views.py

```diff
@@ -65,16 +65,14 @@ def test_tos(self):
         # without validation -> page should ask to accept tos
         response = self.client.get(reverse("core:dashboard"))
         self.assertEqual(response.status_code, 200)
-        self.assertEqual(b"TEST-KEY: ACCEPT_TOS" in response.content, True)
```

Comment:
I'm not sure for this. I think we will have to implement it on the new frontend as well.

Reply:
I have rollback the change for this. It thought it would break the test but it's not the case.